Form-Controls/css/styles.css

```diff
@@ -0,0 +1,102 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Open Sans", sans-serif;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+.container {
+  width: 30rem;
+  margin: 0 auto;
+  margin-top: 5rem;
+  padding: 2rem;
+
+  /* background-color: bisque; */
+}
+
+header {
+  margin-bottom: 2rem;
+}
+
+form {
+  display: grid;
+  justify-items: start;
+  row-gap: 2rem;
+}
+
+.grid--1 {
+  display: grid;
+  justify-content: center;
+  row-gap: 1rem;
+}
+
+label,
+p {
+  font-size: 1.5rem;
+  letter-spacing: 1px;
+  color: #00000090;
+}
+
+input {
+  padding: 1rem 1.5rem;
+  border-radius: 0.8rem;
+  border: 1px solid #c1c1c1;
+  letter-spacing: 0.5px;
+  font-weight: 100;
+  color: #00000090;
+  font-size: 1.5rem;
+  box-shadow: 1px 1px 5px #00000012;
+}
+
+input:focus {
+  outline: none;
+}
+
```

Comment:
@ShayanMahnam! 😊 I’ve added the focus styles as you suggested to enhance accessibility. This will definitely help users navigate the form more easily. Thanks for helpful suggestion🙏